frontend/src/exams/capture-ind-ita-components.js

```diff
@@ -0,0 +1,206 @@
+import Vue from 'vue'
```

Comment:
Why is this a `components.js` file, and not individual exports from a `components` directory?

Reply:
because there is no advantage or disadvantage to either way but then I can stack up the imports together in the curly braces and only have a single` ...from './wherever/whatfile'`